src/main/java/com/commafeed/backend/feed/FeedParser.java

```diff
@@ -145,8 +145,8 @@ private Date getEntryUpdateDate(SyndEntry item) {
 		}
 		return date;
 	}
-
-	private Date validateDate(Date date, boolean nullToNow) {
+	// This function have been changed to protected for testing purposes

```

Comment:
Same as above

Reply:
I will get it as a question for the teacher next class...